src/main/java/uk/gov/hmcts/reform/juddata/camel/binder/JudicialUserProfile.java

```diff
@@ -74,5 +74,7 @@
             message = "date pattern should be " + DATE_TIMESTAMP_FORMAT)
     String extractedDate;
 
+    @DataField(pos = 15, columnName = "object_id")
+    String objectId;
 
```

Comment:
is it mandatory or Non mandatory to have Object ID? if mandatory then need to add JSR. Else have one file with null object id and have one test.

Reply:
Its non mandatory field as couple of entries in CSV are empty rows.

Reply:
All the test files have one empty entry which will test null of object ID.